scripts/dependency_services/common.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+KONG_ENV_FILE=.env
+KONG_ENV_DOWN_FILE=.env.down
```

Comment:
let's use temporary files for those `$(mktemp)`, as the env vars doensn't persist, the file should not either